### PR TITLE
Skip CPU tests on ARM platform

### PR DESCRIPTION
**Pull request by alvoron**
Skip failing CPU tests on ARM platform to enable pre-commit check.
Later these tests will be fixed.

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,13 @@ if (ENABLE_OV_ONNX_FRONTEND)
 else()
     set(EXCLUDED_SOURCE_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/extension ${CMAKE_CURRENT_SOURCE_DIR}/onnx)
 endif()
-
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*|arm64.*|ARM64.*)")
+    list(APPEND EXCLUDED_SOURCE_PATHS
+        ${CMAKE_CURRENT_SOURCE_DIR}/single_layer_tests
+        ${CMAKE_CURRENT_SOURCE_DIR}/subgraph_tests
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/tests/functional/CMakeLists.txt:22]: why all single layer and subgraph tests are excluded?

Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/CMakeLists.txt:22]: It was asked by @dmitry-gorokhov to exclude plugin-specific tests

Comment (ilya-lavrenov) [reply src/plugins/intel_cpu/tests/functional/CMakeLists.txt:22]: ok, I see
I supposed we have excluded all SLT / subgraph tests

File: src/plugins/intel_cpu/tests/functional/CMakeLists.txt [hunk 2]
```diff
@@ -16,7 +16,13 @@ if (ENABLE_OV_ONNX_FRONTEND)
 else()
     set(EXCLUDED_SOURCE_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/extension ${CMAKE_CURRENT_SOURCE_DIR}/onnx)
 endif()
-
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*|arm64.*|ARM64.*)")
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/tests/functional/CMakeLists.txt:19]:
```suggestion
if(ARM OR AARCH64)
```

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 3]
```diff
@@ -201,6 +201,64 @@ std::vector<std::string> disabledTestPatterns() {
     // retVector.emplace_back(R"(.*smoke_LPT.*ReduceMinTransformation.*f32.*)");
 #endif
 
+#if defined(OPENVINO_ARCH_ARM64)
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp:204]:
```suggestion
#if defined(OPENVINO_ARCH_ARM64) || defined(OPENVINO_ARCH_ARM)
```

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**